### ISSUE TITLE
Implementation commitments on RTCEncodingOptions.keyFrame?

### ISSUE THREAD

**Issue by dontcallmedom** [w3c/webrtc-extensions#213]
[`RTCEncodingOptions.keyFrame`](https://w3c.github.io/webrtc-extensions/#dom-rtcencodingoptions-keyframe) (for keyframe generation request) has [reasonable test coverage](https://github.com/web-platform-tests/wpt/blob/master/webrtc/RTCRtpSender-setParameters-keyFrame.html) (although in the wrong directory at the moment), [one passing implementation](https://wpt.fyi/results/webrtc/RTCRtpSender-setParameters-keyFrame.html?label=master&label=experimental&aligned).

@jan-ivar @youennf I could not find any signal of implementation commitment from Firefox or webkit on this; is this something on which you could shed some light?

**Comment by fippo** [comment]
[Firefox position](https://github.com/mozilla/standards-positions/issues/858) and [Webkit position](https://github.com/WebKit/standards-positions/issues/237), no response on either.

I count two passing implementations?

**Comment by dontcallmedom** [comment]
> I count two passing implementations?

if you're thinking Chrome and Edge, they're not considered independent implementation

**Comment by fippo** [comment]
How are "independent implementations" defined then? In this case all four browsers listed use the same underlying implementation (libWebRTCs video engine)

**Comment by dontcallmedom** [comment]
right, and the WG has long acknowledged that this wasn't ideal in terms of independence; but the next best thing is implementations that don't share the same integration layer between the JS API and libwebrtc (which historically has proved productive in identifying bugs / disagreements in the spec)
